projects/packages/paypal-payments/src/legacy/class-simple-payments.php

```diff
@@ -0,0 +1,834 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * Simple Payments lets users embed a PayPal button fully integrated with wpcom to sell products on the site.
+ * This is not a proper module yet, because not all the pieces are in place. Until everything is shipped, it can be turned
+ * into module that can be enabled/disabled.
+ *
+ * @package automattic/jetpack-paypal-payments
+ */
+
+namespace Automattic\Jetpack\Paypal_Payments;
+
+use Automattic\Jetpack\Assets;
+use Automattic\Jetpack\Current_Plan as Jetpack_Plan;
+use Jetpack;
+use Jetpack_Components;
+use Jetpack_Currencies;
```

Comment:
Those Jetpack-the-plugin specific dependencies aren't great. That means you won't be able to use that class on a site that doesn't run the Jetpack plugin. 

I think we'll need to replace them by functionality available in packages, or move those classes to packages to be able to use them.

Reply:
As discussed in Slack, I've added a check in `get_instance()` to bail if this classes don't exist in [302ea6d](https://github.com/Automattic/jetpack/pull/43413/commits/302ea6d996a5ed98290ed5b87eda96a1e200028e).